include/gatekeeper_net.h

```diff
@@ -203,31 +231,15 @@ struct gatekeeper_if {
 	/* Timer to transmit from LLS block to fulfill LACP TX requirement. */
 	struct rte_timer  lacp_timer;
 
-	/* Per-socket ACLs used for classifying IPv6 packets. */
-	struct rte_acl_ctx *ipv6_acls[RTE_MAX_NUMA_NODES];
-
-	/*
-	 * Callback functions for each ACL rule type.
-	 *
-	 * On error, these functions should return a negative value
-	 * and free all packets that have not already been handled.
-	 */
-	acl_cb_func        acl_funcs[GATEKEEPER_IPV6_ACL_MAX];
-
-	/*
-	 * Callback functions for each ACL rule type with
-	 * IPv6 extension headers.
-	 *
-	 * Returning values: 0 means a match and a negative value
-	 * means an error or that there was no match.
-	 */
-	ext_cb_func        ext_funcs[GATEKEEPER_IPV6_ACL_MAX];
-
-	/* Number of ACL types installed in @acl_funcs. */
-	unsigned int       acl_func_count;
+	/* ACLs and associated callback functions for matching packets. */
+	struct acl_state  ipv4_acls;
+	struct acl_state  ipv6_acls;
 
 	/* Whether the EtherType filter can be used on this interface. */
-	bool               hw_filter_eth;
+	bool              hw_filter_eth;
```

Comment:
This space adjustment should be done in the previous patch.

Reply:
If it was done in the previous patch, it would appear to be one space off from all of the ACL fields that come before it.

Since this patch replaces those ACL fields with new fields with different spacing, this patch aligns this field to match up with the new fields.